.devcontainer/Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM osgeo/gdal:ubuntu-small-3.2.1
+FROM osgeo/gdal:ubuntu-small-3.6.1
 
 RUN apt update && apt install -y python3-pip python3-distutils git
 
 RUN adduser --disabled-password --gecos '' vscode
 
 USER vscode
 
-RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_VERSION=1.1.6 python3 -
-
-ENV PATH="~/.poetry/bin:$PATH"
-
```

Comment:
How come we don't have to set the env. variable paths? Is the default that poetry knows where  to install itself? I am still confused about this

Reply:
Or is it just sent in .json now?

Reply:
yeah I think what is in the .json is sufficient for setting up the PATH variable

Reply:
I think I just want to get on the same page about where to specify the PATH and then I can approve this. Tested a bunch of different datasets and the outputs look good